app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -111,6 +113,15 @@ class ProfileEditActivityTest {
   @get:Rule
   val accessibilityTestRule = AccessibilityTestRule()
```

Comment:
Please merge with latest develop as this file `AccessibilityTestRule ` is no longer available in develop code so this should get updated here too.

Reply:
Sorry I did not get it.

Reply:
Can you please guide me through it.

Reply:
If you open your fork code then you can see that your code is behind oppia:develop. https://github.com/shankarpriyank/oppia-android/tree/LabelProfileEditActivity
<img width="878" alt="Screenshot 2022-02-02 at 1 37 15 PM" src="https://user-images.githubusercontent.com/9396084/152116123-55d4e4c8-104e-468e-aeec-fd4243d6d240.png">


So you will need to fix this by fetching the latest develop from oppia and then merging it with your branch. so that it does not say "10 commits behind oppia:develop".

If it still does not make sense then you can meet with your mentor and get it resolved but please make sure you try this on your own first.